SecurityCardManagement/README.md

```diff
@@ -0,0 +1,126 @@
+# SecurityCardManagement Interface
+
+**Status**: Proposed
+
+| Name                        | ID                                                                |
+| :-------------------------- | :---------------------------------------------------------------- |
+| SecurityCardManagement      | 5960c60629709699c0f80756ee9a4074dde26e8f68cddd03bf0507d8eb07915f  |
+
+
+## Description
+
+Allows to manage security card.
+
+## Functions
+
+Since DeBot is a smart contract then all functions work asyncronously by design. It means that they don't return anything despite the fact that all have `returns` value in their specification. Result callback will be called later and it must have arguments defined in `returns` section.
+
+`getBlockHashs` - get H2 and H3 hashes from security card. 
+The browser should call several functions from the sacurity card library to get H2 and H3. The workflow:
+1. get H2 using the getHashOfEncryptedPassword function;
+2. get H3 using getHashOfCommonSecret function.
+
+arguments: 
+
+	answerId: uint32 - function id of result callback
+
+returns: 
+
+	h2: uint256 - H2 hash
+	h3: uint256 - H3 hash
+
+`turnOnWallet` - verify and setup security card initial data.
+
+arguments: 
+
+	answerId: uint32 - function id of result callback	
+	p1      : uint128 - authentication password
+	iv      : uint16 - vector for AES128 CBC initialization for encrypting P1
+	ecs     : uint32 - encrypted common secret
+	pin	: uint16 - security card pin code
+
+returns: 
+
+	pubkey: uint256 - security card public key
+
+`addSigningBox` - add security card signing box to browser. Notify browser to add security card.
```

Comment:
This method might be redundant, since we tend to display the card icon all the time